### ISSUE TITLE
Bug: 21-4140 Question page does not have h1

### ISSUE THREAD

**Issue by jeana-adhoc** [department-of-veterans-affairs/vets-design-system-documentation#4931]
# Bug Report

- [X] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

When doing accessibility tests in 21-4140, I discovered that the unemployment page https://staging.va.gov/disability/eligibility/special-claims/unemployability/employment-questionnaire-form-21-4140/unemployed does not have an h1 on it.

This causes focus to go to the stepper (which is still not a heading)

The question on the page is currently an `h3`

<img width="898" height="699" alt="Image" src="https://github.com/user-attachments/assets/044b2715-a9ad-4d3a-9d95-702827da9d6c" />

## What I expected to happen

I expected the question "Do you certify that you have no employment to report?" to be an `h1` and receive focus when the page loads.

**URL(s) where this bug occurs**

- https://staging.va.gov/disability/eligibility/special-claims/unemployability/employment-questionnaire-form-21-4140/unemployed

**Steps to reproduce:**

1. Go through the form flow
2. Mark that you have no employment to report
3. On the next page, see that the stepper receives the yellow focus ring
4. Inspect the code and see that the question is an `<h3>`

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [X] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [ ] This bug is not blocking any work
- [ ] Other
